### ISSUE TITLE
Quantum Quarry Dimension is Void

### ISSUE THREAD

**Issue by Doomish** [rwtema/extrautilities_old_issues#1263]
Hi, guys i am making skyblock modpack and i have big problem. I use mod named YUnomakegoodmap and its really good but i have big problem. Quantum quarry use dimension -9999 and its void. But my aether is not void. 
Please help i need to solve this. I need this great quarry... 
Thanks for help and sorry for my bad english.

**Comment by ktole1999** [comment]
so what is the issue?

**Comment by Doomish** [comment]
i need to quarry mine. But if quarrys dimension is void, it cant mine.

**Comment by ktole1999** [comment]
are you using the quantum quarry to mine or are you travelling to the dimension. quarry auto mines from the dimension you do not have to place it there.

**Comment by meelock** [comment]
No, what he is saying that there is an "issue" causing the quantum quarry to be defunct (YUnomakenogoodmap should only affect the overworld and the nether,) as the dimension that it "mines" from is empty.

hope this helps!

-meelock

**Comment by ktole1999** [comment]
I was under impression that world was never to be used other than quarry simulating biome to mine from. Might confirm with dev



Sent via the Samsung Galaxy S® 6, an AT&T 4G LTE smartphone


-------- Original message --------

**Comment by blueagledev** [comment]
To my knowledge the quarry creates a dim using the default overworld gen/copies the OW, could you try to check it?

**Comment by InjuredWolf** [comment]
that'll be the issue: it's copying the overworld, which is a void, ergo the quarry dim is a void.

**Comment by Doomish** [comment]
yes this is problem. When i use the lockdown mod, and use template of world its ok. But when i use this YUnomakegoodmap its void. And quarry is void too.

**Comment by Doomish** [comment]
But how can i teleport to this dimension? Because i have not mystcraft with /TPX command

**Comment by blueagledev** [comment]
You could just replace the dimension files with randomly generated OW

**Comment by InjuredWolf** [comment]
Quick question: what does your YUNoMakeGoodMap config look like?

**Comment by Doomish** [comment]
This is the file...
# Configuration file

general {
    # Set to true to enable generation of the end cities.
    B:generateEndCities=true

    # Set to true to enable generation of the nether fortresses.
    B:generateNetherFortress=true

    # Set to true to enable generation of the obsidian 'spikes' in the end.
    B:generateSpikes=true

    # Set to true to force the default world types to be void world. Use with caution.
    B:overrideDefault=false

    # Set the type of platform to create in the overworld, Possible values: 
    #   'STRUCT_NAME' for custom Structure Files, located in /config/YUNoMakeGoodMap/structures/
    #     Example: 'struct:COBBLE_GEN' to use /config/YUNoMakeGoodMap/structures/COBBLE_GEN.nbt
    #   
    #   Default ones provided with this mod:
    #   'SINGLE_GRASS' A single grass block
    #   'TREE' a small oak tree on a grass block
    #   'SKYBLOCK21' For SkyBlock v2.1 platforms
    #   'COBBLE_GEN' Small platform with a pre-built cobble gen
    #   
    #   Other mods can supply platforms as well just need to specify it by using modid:STRUCT_NAME
    #   Which will try and load /assets/modid/structures/STRUCT_NAME.nbt
    S:platformType=grass
}

**Comment by InjuredWolf** [comment]
huh. not sure then. The only thing I would have thought might cause an issue would have been 'overrideDefault', but you have that as false so *shrug*.

**Comment by ktole1999** [comment]
not the first time YUNoMakeGoodMap causes issues from what i have been reading.
